### ISSUE TITLE
Mac M1 support

### ISSUE THREAD

**Issue by rssdev10** [jw3126/ONNXRunTime.jl#22]
Hello, is any way to update binary dependencies https://github.com/jw3126/ONNXRunTime.jl/blob/main/Artifacts.toml
by https://github.com/microsoft/onnxruntime/releases/tag/v1.13.1 ?

For Mac M1 there is a universal archive `onnxruntime-osx-universal2-1.13.1.tgz` with both arm64 and amd64 platforms.

**Comment by jw3126** [comment]
Sure if it is already a tarball you can just add it to Artifacts.toml. If not we need to repackage it. Happy to guide a PR if you want to give it a shot.

**Comment by rssdev10** [comment]
If I understand right, all the binaries are available. And, number of supported architectures in general is much higher compared with the previous version. But, the first step - somebody have to upload these files to `https://github.com/jw3126/ONNXRunTimeArtifacts/releases/download/` .

If you explain me how to do it, I can check ONNXRunTime with both Mac architectures. But I don't have other architectures. And, if you have some script for automation generating the following descriptions and check sums, it would be good to have it too:
```
[[onnxruntime_cpu]]
arch = "x86_64"
git-tree-sha1 = "f07f041b61f199fa853a0e66652633c2d9007478"
lazy = true
libc = "glibc"
os = "linux"

    [[onnxruntime_cpu.download]]
    sha256 = "f386ab80e9d6d41f14ed9e61bff4acc6bf375770691bc3ba883ba0ba3cabca7f"
    url = "https://github.com/jw3126/ONNXRunTimeArtifacts/releases/download/v1.9.0-rc4/onnxruntime-linux-x64-1.9.0.tgz"
 ```

**Comment by jw3126** [comment]
> If I understand right, all the binaries are available. And, number of supported architectures in general is much higher compared with the previous version. But, the first step - somebody have to upload these files to https://github.com/jw3126/ONNXRunTimeArtifacts/releases/download/ .

Repackaging to ONNXRunTimeArtifacts should only be necessary for windows. The other binaries can be taken directly from 
https://github.com/microsoft/onnxruntime/releases/tag/v1.13.1, see also #24.

**Comment by jw3126** [comment]
With #24 merged, it should now be easier for you to add M1 support.

**Comment by jw3126** [comment]
I use this script, in case you need to adapt it: https://github.com/jw3126/ONNXRunTimeArtifacts/blob/main/script.jl

**Comment by rssdev10** [comment]
Thanks, but a minor fix for the script:

```julia
    (
    artifact_name="onnxruntime_cpu",
    download_name="onnxruntime-osx-universal2-$version.tgz",
    platform=Pkg.Artifacts.Platform("x86_64", "macos"),
    version=version,
   ),
    (
    artifact_name="onnxruntime_cpu",
    download_name="onnxruntime-osx-universal2-$version.tgz",
    platform=Pkg.Artifacts.Platform("aarch64", "macos"),
    version=version,
   ),
```

**Comment by rssdev10** [comment]
and, with #25 that works on M1.

**Comment by jw3126** [comment]
fixed by #25 if some issues remain, please reopen.
